pypixplore/remote.py

```diff
@@ -56,3 +61,33 @@ def get_well_maintained(self):
         Get packages which have had at least one release in the last six months, sorted by most recently updated
         """
         raise NotImplementedError
+
+    def count_releases(self, package_name, time_days):
+        json = self._get_JSON(package_name)
+        if json == []:
+            return(0)
+        keys = json["releases"].keys()
+        count = 0
+        for key in keys:
+            if json["releases"][key] == []:
+                break
+            date = json["releases"][key][0]["upload_time"]
+            date = datetime.datetime(int(date[0:4]), int(date[5:7]), int(date[8:10]), 0, 0, 0)
```

Comment:
Here you should use the `strptime` function from the datetime library to parse the date instead of building the date by hand.